src/Promitor.Tests.Unit/Validation/Scraper/Metrics/Sinks/StatsDMetricSinkValidationStepTests.cs

```diff
@@ -87,6 +88,38 @@ public void Validate_StatsDWithoutHost_Fails(string host)
             PromitorAssert.ValidationFailed(validationResult);
         }
 
+        [Theory]        
+        [InlineData(null)]
+        public void Validate_StatsDWithGenevaFormatWithoutGenevaConfiguration_Fails(GenevaConfiguration genevaConfiguration)
+        {
+            // Arrange
+            var runtimeConfiguration = CreateRuntimeConfiguration();
+            runtimeConfiguration.Value.MetricSinks.Statsd.MetricFormat = StatsdFormatterTypesEnum.Geneva;
+            runtimeConfiguration.Value.MetricSinks.Statsd.Geneva = genevaConfiguration;
+
+            // Act
+            var azureAuthenticationValidationStep = new StatsDMetricSinkValidationStep(runtimeConfiguration, NullLogger<StatsDMetricSinkValidationStep>.Instance);
+            var validationResult = azureAuthenticationValidationStep.Run();
+
+            // Assert
+            PromitorAssert.ValidationFailed(validationResult);
+        }
+
+        [Fact]
+        public void Validate_StatsDWithGenevaFormatWithGenevaConfiguration_Success()
+        {
+            // Arrange
+            var runtimeConfiguration = CreateRuntimeConfiguration();
+            runtimeConfiguration.Value.MetricSinks.Statsd.MetricFormat = StatsdFormatterTypesEnum.Geneva;
```

Comment:
Let's explicitly add a geneva configuration, because with the current config it should fail (unless CreateRuntimeConfiguration configures it)

Reply:
I've added a configuration for Geneva in GenerateStatsdSinkConfiguration, should I remove it?

Reply:
It's OK